src/components/scenes/GettingStartedScene.tsx

```diff
@@ -0,0 +1,349 @@
+              <MainButton type="escape" onPress={handlePressSignIn} label={s.strings.getting_started_button_sign_in} />
+            </Space>
+          </SectionCoverAnimated>
+        </Container>
+      </SwipeOffsetDetector>
+    </SceneWrapper>
+  )
+}
+
+// -----------------------------------------------------------------------------
+// Local Components
+// -----------------------------------------------------------------------------
+
+const Container = styled(View)(_ => ({
+  flex: 1
+}))
+
+//
+// Skip Button
+//
+
+const SkipButton = styled(Animated.View)<{ swipeOffset: SharedValue<number> }>(props =>
+  useAnimatedStyle(() => {
+    return {
+      opacity: interpolate(props.swipeOffset.value, [0, 1], [0, 1])
+    }
+  })
+)
+
+//
+// Hero
+//
+
+const HeroContainer = styled(Animated.View)(_ => ({
+  flex: 1,
+  alignItems: 'center'
+}))
+
+const WelcomeHero = styled(Animated.View)<{ swipeOffset: SharedValue<number> }>(props => [
+  {
+    alignItems: 'center',
+    justifyContent: 'center',
+    flex: 1
+  },
+  useAnimatedStyle(() => ({
+    opacity: interpolate(props.swipeOffset.value, [0, 0.5], [1, 0]),
+    transform: [{ scale: interpolate(props.swipeOffset.value, [0, 1], [1, 0], Extrapolation.CLAMP) }]
+  }))
+])
+
+const WelcomeHeroTitle = styled(EdgeText)(props => ({
+  fontSize: props.theme.rem(2.25),
+  paddingVertical: props.theme.rem(1),
+  textAlign: 'center'
+}))
+const WelcomeHeroMessage = styled(EdgeText)(props => ({
+  fontSize: props.theme.rem(0.78),
+  paddingVertical: props.theme.rem(1),
+  textAlign: 'center'
+}))
+const WelcomeHeroPrompt = styled(EdgeText)(props => ({
+  fontSize: props.theme.rem(0.75),
+  fontFamily: props.theme.fontFaceBold,
+  textAlign: 'center'
+}))
+
+const HeroItem = styled(Animated.View)<{ swipeOffset: SharedValue<number>; itemIndex: number }>(props => {
+  const { swipeOffset, itemIndex } = props
+  const isFirstItem = itemIndex === 1
+  const { width: screenWidth } = useSafeAreaFrame()
+  const translateWidth = screenWidth
+  return [
+    {
+      alignItems: 'center',
+      aspectRatio: 1,
+      padding: props.theme.rem(2),
+      position: 'absolute',
+      height: '100%',
+      width: '100%'
+    },
+    useAnimatedStyle(() => {
+      const opacity = interpolate(swipeOffset.value, [itemIndex - 1, itemIndex, itemIndex + 1], [0, 1, 0], Extrapolation.CLAMP)
+      const scale = interpolate(swipeOffset.value, [itemIndex - 1, itemIndex, itemIndex + 1], [0.3, 1, 0.3])
+      const translateX = interpolate(swipeOffset.value, [itemIndex - 1, itemIndex, itemIndex + 1], [isFirstItem ? 0 : translateWidth, 0, -translateWidth])
+      return {
+        opacity,
+        transform: [{ translateX }, { scale }]
+      }
+    })
+  ]
+})
+
+const HeroImageContainer = styled(View)(_ => ({
+  alignItems: 'stretch',
+  aspectRatio: 1,
+  backgroundColor: 'white',
+  borderRadius: 1000,
+  maxHeight: '100%',
+  overflow: 'hidden',
+  width: '100%'
+}))
```

Comment:
Since this guy exists to render a white circle, maybe call him `HeroImageCircle`.

I had a long thing typed up about a simpler way to position the image, but then I realized that you were actually doing it this way to get a circle, not to control the image layout.

Reply:
If the design was a rounded rectangle, then we'd have to rename it to HeroImageRoundedRectangle. But if we keep it HeroImageContainer, no renaming is required because it will always remain the container of the hero image semantically. Does this seem appropriate?